runtimes/go/appruntime/apisdk/api/transport/eh2c.go

```diff
@@ -0,0 +1,35 @@
+package transport
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+	"net/http"
+
+	"golang.org/x/net/http2"
+)
+
+func NewH2CTransport() http.RoundTripper {
```

Comment:
Maybe accept the non-h2c transport to use as an argument instead of implicitly using `http.DefaultTransport`?